.gitignore

```diff
@@ -198,6 +195,30 @@ vendor/webmozart/
 vendor/misantron/
 upload/templates/
 assets/packages/ckeditor/samples
+vendor/composer/pcre/
```

Comment:
These packages are actually committed to the repository so we should not add them to .gitignore. 

You simply have to remember not to commit these files to the repository.

Reply:
I reverted .gitignore.